src/utils.cpp

```diff
@@ -1069,6 +1069,11 @@ std::string utils::make_title(const std::string& const_url) {
 	if (title.at(0)>= 'a' && title.at(0)<= 'z') {
 		title[0] -= 'a' - 'A';
 	}
+	//strip .htm or .html from title
```

Comment:
Better move this up and place it on line 1065. This will keep the flow of the function straightforward: it gradually trims the URL down to the title.